### ISSUE TITLE
[photospline] Backport fixes to package-test branch (Trac #1726)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1718]
The standalone package-test branch, for use outside an IceTray environment, doesn't have the latest bug fixes and Python3 updates.

Migrated from https://code.icecube.wisc.edu/ticket/1726
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:47", 
    "description": "The standalone package-test branch, for use outside an IceTray environment, doesn't have the latest bug fixes and Python3 updates.\n", 
    "reporter": "jkelley", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067167842669", 
    "component": "combo reconstruction", 
    "summary": "[photospline] Backport fixes to package-test branch", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-06-03T14:43:19", 
    "milestone": "", 
    "owner": "jkelley", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160606T15:17:29: 
* jkelley changed status from "new" to "closed"
* jkelley commented:

> In changeset 146882:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="146882"
> Backport changes from trunk as of June 2016.  Includes a number of bug fixes and Python 3 updates.  Fixes #1718.
> ```


* jkelley changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:47: 
* olivas changed milestone from "June 2016" to ""
* olivas commented:

> Milestone June 2016 deleted
